src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,6 +302,58 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
     );
   });
+
+  test("Verify that the Premium upsell screen is displayed correctly - overview card", async ({
+    dashboardPage,
+    // purchasePage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463625",
+    });
+
+    //checking that the user can reach upsell page
+    await dashboardPage.goToDashboard();
+    const upsellButton = page.locator("section > div a");
+    await expect(upsellButton).toBeVisible();
+    await upsellButton.click();
+    await page.waitForURL(/.*\/fix\/.*\/view-data-brokers\/?/);
+    const removeThemButton = page.getByText("Remove them for me");
+    await removeThemButton.click();
+
+    //checking the bullet points
+    await page.waitForURL(/.*\/fix\/.*\/automatic-remove\/?/);
+    const ulElement = page.locator("div > strong + ul");
+    expect(await ulElement.isVisible()).toBeTruthy();
+
+    const expectedItems = [
+      "Monthly scan of ⁨190⁩ data broker sites that may be selling your personal info",
+      "Automatic data removal from sites that are selling your personal info",
+      "Guided experience through high risk data breaches that require manual steps",
+      "Continuous monitoring for new exposures",
+      "Alerts when your data has been breached",
+    ];
+
+    for (const itemText of expectedItems) {
+      const liElement = page.locator(`ul li:has-text("${itemText}")`).first();
+      await expect(liElement).toBeVisible();
+    }
+
+    //testing that toggles work
+    const label0 = page.locator("div > label").nth(0);
+    const label1 = page.locator("div > label").nth(1);
+
+    await label0.click();
+    const price0 = page.locator("div > strong + span").textContent();
+    const plan0 = page.locator("div > strong + span + a").textContent();
+    await label1.click();
+    const price1 = page.locator("div > strong + span").textContent();
+    const plan1 = page.locator("div > strong + span + a").textContent();
+    const areTextsChanging = price0 !== price1 && plan0 !== plan1;
+    expect(areTextsChanging).toBeTruthy();
```

Comment:
not sure if this might be simpler w/ two separate asserts something like this:
```js
expect(price0).not.toEqual(price1);
expect(plan0).not.toEqual(plan1);
```
If nothing else, will give you slightly more helpful error messaging vs "expected true, received false".